engine/apps/telegram/tasks.py

```diff
@@ -119,7 +119,14 @@ def send_link_to_channel_message_or_fallback_to_full_incident(
 @ignore_bot_deleted
 def send_log_and_actions_message(self, channel_chat_id, group_chat_id, channel_message_id, reply_to_message_id):
     with OkToRetry(task=self, exc=TelegramMessage.DoesNotExist, num_retries=5):
-        channel_message = TelegramMessage.objects.get(chat_id=channel_chat_id, message_id=channel_message_id)
+        try:
+            channel_message = TelegramMessage.objects.get(chat_id=channel_chat_id, message_id=channel_message_id)
+        except TelegramMessage.DoesNotExist:
```

Comment:
Here we use `OkToRetry` to gracefully retry 5 times in case we can't find a Telegram message in the DB, because there are cases when the message was already forwarded to discussion group, but wasn't created in the database yet.

I think we can do something like:

```
if self.request.retries <= 5:
    # retry the task gracefully
else:
    # log and return
``` 

There's an example of using this approach in `apps.telegram.tasks.send_link_to_channel_message_or_fallback_to_full_incident`.